app/Http/Controllers/Backend/UserController.php

```diff
@@ -28,17 +31,32 @@ public function index()
      */
     public function create()
     {
-        //
+        return view('admin.users.create');
     }
 
     /**
      * Store a newly created resource in storage.
      *
+     * @param \Illuminate\Http\UserRequest $request User request
+     *
      * @return \Illuminate\Http\Response
      */
-    public function store()
+    public function store(UserRequest $request)
     {
-        //
+        $data = $request->all();
+        $data['admin_user_id'] = Auth::guard('admin')->user()->id;
+        $img = "";
```

Comment:
[rank B] why did you init this variable as "" and assign it as null when it cannot upload successful?